addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -318,6 +338,16 @@ function factory(dependencies) {
         managerVisual: attr({
             related: 'manager.visual',
         }),
+        /**
+         * This field handle the "order" (index) of the visible chatWindow inside the UI.
+         */
+        visibleIndex: attr({
+            compute: '_computeVisibleIndex',
+            dependencies: [
+                'manager',
+                'managerVisual',
+            ],
+        }),
```

Comment:
Here are interesting details to add in the comment, especially to understand the tests:

Using LTR, the right-most chat window has index 0, and the number is incrementing from right to left.
Using RTL, the left-most chat window has index 0, and the number is incrementing from left to right.